lib/modules/filteReddit/postCases/Score.js

```diff
@@ -21,6 +21,9 @@ export class Score extends Case {
 
 	evaluate(thing: *) {
 		const score = thing.getScore();
-		return isNaN(score) ? false : numericalCompare(this.value.op, score, this.value.val);
+		if (typeof score !== "number" || isNaN(score)) {
```

Comment:
yet somehow everything gets filtered anyway. filtering has gotten beyond the grasp of my feeble pres-RES-5 mind.

Reply:
Note that even if this returns `false` the post thing is hidden _when_ the filter's state is inverted (i.e. only display posts which returns `true`). https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/9b4a2fa91d42e6b4b7fed50da6dddb6ba7f6028c/lib/modules/filteReddit/Filter.js#L133-L134 `Case#evaluate` should probably just throw an error instead when it's not able to evaluate a post.

Another way to solve this could be to just disable cases until Thing is built out.
https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/9b4a2fa91d42e6b4b7fed50da6dddb6ba7f6028c/lib/modules/filteReddit/postCases/IsVisited.js#L12-L14